lib/scrabble.rb

```diff
@@ -10,4 +10,49 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+    score = 0
+    if word != nil
+      split_letters(word).each do |letter|
+        score += point_values[letter]
+      end
+    end
+    score
+  end
+
+  def score_with_multipliers(word, multipliers, word_mult = 1)
```

Comment:
This looks really good - I think there might be some room for refactoring into some helper methods, maybe the 7-letter bonus could be some sort of helper?